Added ambiguity_threshold to fallback actions page (#5857)
RasaHQ/rasa: changelog/4441.doc.rst

> Added ambiguity_threshold in fallback actions page.
> Previously ambiguity_threshold was missing in fallback actions page.

Reviewer: ```suggestion
Added documentation on ``ambiguity_threshold`` parameter in Fallback Actions page.
```
